code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -522,7 +525,7 @@
 	if(istype(src,/obj/item/device/dogborg/sleeper/compactor/analyzer))
```

Comment:
The sort of general lack of polymorphism in this file is becoming a problem. `istype(src` should never be necessary.